management/server/http/policies_handler.go

```diff
@@ -100,26 +105,49 @@ func (h *Policies) UpdatePolicy(w http.ResponseWriter, r *http.Request) {
 	if req.Rules != nil {
```

Comment:
we should allow empty rules in request and return an error

Reply:
Fixed